src/backend/context/NotesContext.js

```diff
@@ -0,0 +1,47 @@
+import {
```

Comment:
Why are we having the context related files in the backend folder?Since this file is involved with working on the frontend you can keep it outside the backend!!

Reply:
Thank you for the suggestions